src/freebsd/process.rs

```diff
@@ -207,6 +212,11 @@ pub(crate) unsafe fn get_process_data(
     };
     let status = ProcessStatus::from(kproc.ki_stat);
 
+    // from FreeBSD source /bin/ps/print.c
+    let accum_cpu_usage = (kproc.ki_runtime as f64 / 1000000.0) as f32
+        + kproc.ki_childtime.tv_sec as f32
+        + kproc.ki_childtime.tv_usec as f32 / 1000000.0;
```

Comment:
I don't think this is quite right. Here is the [relevant code from FreeBSD](https://github.com/freebsd/freebsd-src/blob/f0e59b6903c881859d7f4eaf269b2336b774c7bc/bin/ps/print.c#L542C1-L547C3):

```c
	secs = k->ki_p->ki_runtime / 1000000;
	psecs = k->ki_p->ki_runtime % 1000000;
	if (sumrusage) {
		secs += k->ki_p->ki_childtime.tv_sec;
		psecs += k->ki_p->ki_childtime.tv_usec;
	}
```

It is clear that child time is only added if `sumrusage` is true. It is set by the `-S` switch:

```
   -S      Change  the way the process times, namely cputime, systime, and
	       usertime, are calculated	by  summing  all  exited  children  to
	       their parent process.
```

Reply:
Ah, you're right, I thought this needed to account for the child's time, but that is handled separately.